examples/deepspeed/cifar10_cpu_offloading/zero_stages_3_no_offload.yaml

```diff
@@ -0,0 +1,37 @@
+name: OOM error
+debug: false
+#profiling:
+#    enabled: true
+#    begin_on_batch: 1
+#    end_after_batch: 1000
+#    sync_timings: false
+hyperparameters:
+  deepspeed_config: ds_config_no_offload.json
+environment:
```

Comment:
let's use this docker image: determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-deepspeed-0.7.0-gpu-9119094

then we shouldn't need to install pdsh, torch, torchvision, or deepspeed as part of startup hook

Reply:
done. startup-hook and requirements.txt removed - not needed anymore.